src/com/egs/suzy/task1/TestCountHellos.java

```diff
@@ -0,0 +1,16 @@
+import java.io.File;
+import java.io.IOException;
+
+public class TestCountHellos {
+    public static void main(String[] args) throws IOException {
+        File hello = new File("hello.txt");
+        File result = new File("result.txt");
+        if(result.exists()){
+            result.delete();
+            result.createNewFile();
```

Comment:
I think creating new file is not necessary

Reply:
I wanted to have clear result file for each check and thought recreation of the new file is faster than clearing the content